packages/edit-post/src/editor.native.js

```diff
@@ -135,10 +135,8 @@ class Editor extends Component {
 		if ( ! prevProps.isReady && this.props.isReady ) {
 			const blocks = this.props.getEditorBlocks();
 			const isUnsupportedBlock = ( { name } ) => name === getUnregisteredTypeHandlerName();
-			const unsupportedBlocks = blocks.filter( isUnsupportedBlock );
-			const hasUnsupportedBlocks = ! isEmpty( unsupportedBlocks );
-
-			RNReactNativeGutenbergBridge.editorDidMount( hasUnsupportedBlocks );
+			const unsupportedBlocksName = blocks.filter( isUnsupportedBlock ).map( ( block ) => block.attributes.originalName );
+			RNReactNativeGutenbergBridge.editorDidMount( unsupportedBlocksName );
```

Comment:
Very small detail:
For consistency we could change the name from `unsupportedBlocksName` to `unsupportedBlockNames`

Reply:
Right. Renamed!